controllers/dscinitialization/dscinitialization_test.go

```diff
@@ -102,6 +104,16 @@ var _ = Describe("DataScienceCluster initialization", func() {
 			expectedConfigmapData := map[string]string{"namespace": applicationNamespace}
 			Expect(foundConfigMap.Data).To(Equal(expectedConfigmapData))
 		})
+
+		It("Should create the trusted CA Bundle configmap", func() {
+			// then
+			foundConfigMap := &corev1.ConfigMap{}
+			Eventually(objectExists(caBundleConfigmapName, applicationNamespace, foundConfigMap), timeout, interval).Should(BeTrue())
+			Expect(foundConfigMap.Name).To(Equal(caBundleConfigmapName))
+			Expect(foundConfigMap.Namespace).To(Equal(applicationNamespace))
+			expectedConfigmapData := map[string]string{caBundleDataField: ""}
+			Expect(foundConfigMap.Data).To(Equal(expectedConfigmapData))
+		})
```

Comment:
small suggestion that improves readability

```suggestion
		It("Should create the trusted CA Bundle configmap", func() {
			// then
			foundConfigMap := &corev1.ConfigMap{}
			Eventually(objectExists(caBundleConfigmapName, applicationNamespace, foundConfigMap)).
				WithTimeout(timeout).
				WithPolling(interval).
				Should(BeTrue())
			
			Expect(foundConfigMap.Name).To(Equal(caBundleConfigmapName))
			Expect(foundConfigMap.Namespace).To(Equal(applicationNamespace))
			Expect(foundConfigMap.Data).To(HaveKeyWithValue(caBundleDataField, ""))
		})
```